cloud_cluster_node:
  ros__parameters:
    publish_frequency: 10.0  # Hz
    cluster_tolerance: 0.15 # cm
    min_cluster_size: 100
    max_cluster_size: 25000
    voxel_leaf_size: 0.03  # reduce voxel size (cm), for better performance
    frame_id: "odom"  # refrence frame for node graph
    target_frame: "odom"
    semantic_pointcloud_topic: "/yoloe/score_cloud"  # topic for point cloud data
    publish_bounding_boxes: false